src/pages/AccountSettings/tabs/OrgUploadToken/TokenlessSection/TokenlessSection.test.tsx

```diff
@@ -1,139 +1,241 @@
-import { render, screen } from '@testing-library/react'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { render, screen, waitFor } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
+import { graphql, HttpResponse } from 'msw2'
+import { setupServer } from 'msw2/node'
 import { MemoryRouter, Route } from 'react-router-dom'
 
+import { useAddNotification } from 'services/toastNotification'
+import { useFlags } from 'shared/featureFlags'
+
 import TokenlessSection from './TokenlessSection'
 
-const Wrapper: React.FC<React.PropsWithChildren> = ({ children }) => (
-  <MemoryRouter initialEntries={['/account/github/codecov/org-upload-token']}>
-    <Route path="/account/:provider/:owner/org-upload-token">{children}</Route>
-  </MemoryRouter>
+const mocks = vi.hoisted(() => ({
+  useAddNotification: vi.fn(),
+}))
+
+vi.mock('shared/featureFlags')
+vi.mock('services/toastNotification', async () => {
+  const actual = await vi.importActual('services/toastNotification')
+  return {
+    ...actual,
+    useAddNotification: mocks.useAddNotification,
+  }
+})
+
+const mockedUseFlags = useFlags as jest.Mock
```

Comment:
I think we may need to swap these to vi.mock() similar to this

https://github.com/codecov/gazebo/blob/01a7a7cf16d3b19e6b9e7f34096c0120ca6aed51/src/pages/AnalyticsPage/Tabs/Tabs.test.tsx#L21-L22

Reply:
good catch